projects/plugins/crm/src/automation/commons/conditions/contacts/class-contact-status-changed.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Jetpack CRM Automation Contact_Status_Changed condition.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Conditions;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Exception;
+use Automattic\Jetpack\CRM\Automation\Automation_Logger;
+use Automattic\Jetpack\CRM\Automation\Base_Condition;
+
+/**
+ * Contact_Status_Changed condition class.
+ *
+ * @since $$next-version$$
+ */
+class Contact_Status_Changed extends Base_Condition {
+
+	/**
+	 * The Automation logger.
+	 *
```

Comment:
It seems we have to use the WPCS expected format ([comment from other PR](https://github.com/Automattic/jetpack/pull/32185#discussion_r1284368022))